package.json

```diff
@@ -63,5 +65,16 @@
   "dependencies": {
     "elly": "^1.1.10",
     "svgson": "^3.1.0"
-  }
+  },
+  "browserslist": [
+    "last 2 Chrome versions",
+    "last 2 ChromeAndroid versions",
+    "last 2 Firefox versions",
+    "last 2 FirefoxAndroid versions",
+    "last 2 Safari versions",
+    "last 2 iOS versions",
+    "last 2 Edge versions",
+    "last 2 Opera versions",
+    "last 2 OperaMobile versions"
```

Comment:
What is the purpose of this list? Since we don't have a list of supported browsers, why not just a single one for dev purposes?

Reply:
@thisismydesign We don't want Parcel to use Babel to translate our async/await calls (since we are probably using modern browsers, we would want to use this translate for production.) As such, we add this to package.json, this will target all evergreen browsers, but we can just put the one that we are using.

Reply:
Sounds good. I'd recommend removing at least mobile browsers.

Reply:
There are two things I couldn't understand.

> We don't want Parcel to use Babel to translate our async/await calls 

According to their [documentation](https://parceljs.org/transforms.html#default-babel-transforms), this is the default behavior, isn't this? Does it ignore `.babelrc` if `browserslist` is defined?

> we would want to use this translate for production

What do you mean?

Reply:
> According to their documentation, this is the default behavior, isn't this? Does it ignore .babelrc if browserslist is defined?

The problem is Parcel by default support ie11 [which does not support native async/await](https://caniuse.com/#feat=async-functions), and with [browserlist](https://github.com/browserslist/browserslist) we can tell parcel which browsers that we want Babel, and PostCSS target. 
I don't know how this affects other .babelrc settings.

> What do you mean?

If we want to use async/await native and avoid configuring .babelrc, using babel-polyfill, babel-runtime in dev this is a way for that, and in production, it depends on our audience, which needs to come from our analytics, but IMO if we support the latest two [evergreen](https://www.techopedia.com/definition/31094/evergreen-browser) browser it will cover nearly 100% of our users.

Reply:
Got it, thanks for the detailed answer.